src/style.css

```
@@ -0,0 +1,44 @@
+body {
+  margin: 0;
+}
+
+.stars--0,
+.stars--1,
+.stars--2,
+.stars--3,
+.stars--4,
+.stars--5 {
+  display: flex;
+}
```

You are misusing the modifier: class `stars` is needed here
```suggestion
.stars {
  display: flex;
}
```